lxd/db/schema/schema.go

```diff
@@ -164,6 +165,19 @@ func ensureUpdatesAreApplied(tx *sql.Tx, updates []Update, hook Hook) error {
 		return fmt.Errorf("failed to fetch update versions: %v", err)
 	}
 
+	// Fix bad upgrade code between 30 and 32
```

Comment:
FWIW, in this kind of situations I find it useful for future readers (including ourselves) to spend a bit more words in providing context, e.g. mention that we got 3 cases of this and that update 31 is now effectively a no-op. Alternatively filing a GH issue with this info and referencing the number in the comment would be even better. YMMV :)